server/src/test/java/com/linecorp/centraldogma/server/MetricsTest.java

```diff
@@ -55,23 +59,27 @@ void metrics() {
         assertThat(((CompositeMeterRegistry) meterRegistry).getRegistries())
                 .hasAtLeastOneElementOfType(PrometheusMeterRegistry.class);
 
-        AggregatedHttpResponse res = dogma.httpClient().get("/monitor/metrics").aggregate().join();
+        final AggregatedHttpResponse res = dogma.httpClient().get("/monitor/metrics").aggregate().join();
         String content = res.contentUtf8();
         assertThat(res.status()).isEqualTo(HttpStatus.OK);
         assertThat(res.contentType()).isEqualTo(MediaType.parse(TextFormat.CONTENT_TYPE_004));
         assertThat(content).isNotEmpty();
         assertThat(content).doesNotContain(
                 "com.linecorp.centraldogma.server.internal.api.WatchContentServiceV1");
 
-        dogma.client()
-             .forRepo("foo", "bar")
-             .watch(Query.ofJson("/foo.json"))
-             .timeoutMillis(100)
-             .errorOnEntryNotFound(false)
-             .start()
-             .join();
-        res = dogma.httpClient().get("/monitor/metrics").aggregate().join();
-        content = res.contentUtf8();
+        final WatchRequest<JsonNode> jsonNodeWatchRequest = dogma.client()
+                                                                 .forRepo("foo", "bar")
+                                                                 .watch(Query.ofJson("/foo.json"))
+                                                                 .timeoutMillis(100)
+                                                                 .errorOnEntryNotFound(false);
+        jsonNodeWatchRequest.start().join();
+        content = dogma.httpClient().get("/monitor/metrics").aggregate().join().contentUtf8();
         assertThat(content).contains("com.linecorp.centraldogma.server.internal.api.WatchContentServiceV1");
+        assertThat(content).doesNotContain("init_revisions");
+
+        // Trigger old revision recording
+        jsonNodeWatchRequest.start(Revision.INIT).join();
+        content = dogma.httpClient().get("/monitor/metrics").aggregate().join().contentUtf8();
+        assertThat(content).contains("init_revisions");
```

Comment:
Can we also add a similar test for `old_revisions`? Should check if the meters with `init=true` and `init=false` both exists.

Reply:
All addressed. PTAL. 😉